blog/content/post/2020-03-28-trenchboot-nlnet-introduction.md

```diff
@@ -47,8 +47,9 @@ about the project.
 ## Our goal
 
 Our goal is twofold:
-1. Implement DRTM with TrenchBoot for AMD processors.
-2. Create a test environment with test suites for the community.
+
+* Implement DRTM with TrenchBoot for AMD processors.
+* Create a test environment with test suites for the community.
```

Comment:
@piotr-kleins right now it is better to use unordered list due to the https://github.com/3mdeb/news-and-ideas/issues/188 issue